app/assets/javascripts/darkswarm/controllers/products_controller.js.coffee

```diff
@@ -50,7 +50,7 @@ Darkswarm.controller "ProductsCtrl", ($scope, $filter, $rootScope, Products, Ord
   $scope.appliedPropertiesList = ->
     $scope.activeProperties.map( (property_id) ->
       Properties.properties_by_id[property_id].name
-    ).join(" & ") if $scope.activeProperties?
+    ).join(" OR ") if $scope.activeProperties?
```

Comment:
We would also need to make it possible to translate this string to all supported languages. https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29 has all the information you may need.

Basically, we need to add a new key in `config/locales/en.yml`, the default language, and then refer to it in here. This way we'll use the appropriate copy for each instance.

Reply:
done! (checked in the browser - working fine for me)